src/components/heading/heading.style.js

```diff
@@ -64,4 +61,71 @@ StyledHeadingIcon.defaultProps = {
   theme: baseTheme,
 };
 
-export { StyledHeadingIcon, StyledHeading };
+const StyledSubHeader = styled.div`
+  margin-top: 5px;
+`;
+
+StyledSubHeader.defaultProps = {
+  theme: baseTheme,
+};
+
+const StyledHeader = styled.div`
+  position: relative;
+`;
+
+StyledHeader.defaultProps = {
+  theme: baseTheme,
+};
+
+const StyledDivider = styled.hr`
```

Comment:
Why aren't we using our `Hr` component here?

Reply:
In fact this `StyledDivider` isn't needed at all, I'll comment above with the solution

Reply:
replaced with Hr